### DISCUSSION TITLE
Presentation on 22MAY: Cloud Related Service Level Agreements

### DISCUSSION THREAD

**Discussion by brianrandallfox** [GSA-TTS/federal-platform-engineering-cop#9] [General]
A conversation thread here!

**Comment by brianrandallfox** [comment]
[Cloud Service Level Objectives and Agreements.pptx](https://github.com/GSA-TTS/federal-platform-engineering-cop/files/15405334/Cloud.Service.Level.Objectives.and.Agreements.pptx)
These are the slides from today's presentation.  Many thanks to Karin!

**Comment by brianrandallfox** [comment]
Here is the Service Level Agreement (SLA) template that Karin shared today.
[Platform Engineering - Service Level Agreement Template.docx](https://github.com/GSA-TTS/federal-platform-engineering-cop/files/15405361/Platform.Engineering.-.Service.Level.Agreement.Template.docx)

**Comment by pburkholder** [comment]
Only tangentially related, but Karin mentioned the GAO and IG, and the DOI IG has some folks doing technically interesting work. 

Example 1: The IG worked with DOI IT to [obtain the DOI's hashed passwords, then proceeded to crack them](https://www.doioig.gov/sites/default/files/2021-migration/Final%20Inspection%20Report_DOI%20Password_Public.pdf) and report back on how many weak passwords there were.

Example 2: The [IG investigated DLP](https://www.doioig.gov/sites/default/files/2021-migration/Final%20Evaluation%20Report_Security%20of%20Data%20Stored%20in%20Cloud_Public.pdf) from a cloud by using a publicly available web application to generate our fictitious
sensitive PII, and then attempting to exfiltrate it. 

So it's a refreshing to see an IG willing to roll up their sleeves and get technical....
